config/scout.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Default Search Engine
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default search connection that gets used while
+    | using Laravel Scout. This connection is used when syncing all models
+    | to the search service. You should adjust this based on your needs.
+    |
+    | Supported: "algolia", "null"
+    |
+    */
+
+    'driver' => env('SCOUT_DRIVER', 'algolia'),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Index Prefix
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify a prefix that will be applied to all search index
+    | names used by Scout. This prefix may be useful if you have multiple
+    | "tenants" or applications sharing the same search infrastructure.
+    |
+    */
+
+    'prefix' => env('SCOUT_PREFIX', ''),
```

Comment:
Since we'll have multiple apps in here eventually, we'll probably want to set this right? I wonder if it'd make sense to have our [Terraform `heroku_app` module](https://github.com/DoSomething/infrastructure/blob/master/components/heroku_app/main.tf) set an `APP_NAME` environment variable that we can just read in here...

Reply:
I didn't set anything yet because I needed to research this a bit more. I'm not sure if we ever want Contentful to add data to indexed campaigns if it means we need to then specify the rogue specific prefix or whatnot. I just wasn't sure and wanted to review this option a bit more.